client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -22,7 +22,7 @@
  * to only one reader.
  *
  * The readers in the group may change over time. Readers are added to the group by calling
- * {@link ClientFactory#createReader(String, String, Serializer, ReaderConfig)} and are removed by
+ * {@link io.pravega.client.EventStreamClientFactory#createReader(String, String, Serializer, ReaderConfig)} and are removed by
```

Comment:
Exceeds max line length. Please wrap the line.

Reply:
done